dox/new_header.html

```diff
@@ -51,12 +51,5 @@
  </tr>
  </tbody>
 </table>
-</div>
```

Comment:
Please restore this &lt;/div&gt;.  If I compare the current version of this file against the default generated header html file, this line is missing.  It matches the &lt;div&gt; just before the &lt;table&gt; that was just closed.

Reply:
The double up of "Bill" looks clutterd and repetitive. If this solution is awkward, do you have an alternative suggeation?